doc/operator-user-guide.md

```diff
@@ -23,6 +23,7 @@
 * [3scale installation Backup and Restore using the operator (in *TechPreview*)](operator-backup-and-restore.md)
 * [Application Capabilities (in *TechPreview*)](operator-application-capabilities.md)
 * [APIManager CRD reference](apimanager-reference.md)
+* [AWS STS Support](#aws-sts-support)
```

Comment:
Add STS user guide in the `S3 Filestorage Installation` section

Reply:
@eguzki , do you mean - to move "AWS STS Suppor" section to be under S3 Filestorage Installation section ?

Reply:
Done. doc updated. Thank you

Reply:
it is still there